### ISSUE TITLE
"Column definition lwt_indicator does not match any column in the query selection" during schema changes

### ISSUE THREAD

**Issue by KnifeyMoloko** [scylladb/scylladb#10192]
*Installation details*
Kernel version: `5.11.0-1028-aws`
Scylla version (or git commit hash): `5.1.dev-0.20220304.eff5076dd5d9 with build-id a788ea8b5f4d4cbbcf8711974791711b2b88924f`
Cluster size: 4 nodes (i3.2xlarge)
Scylla running with shards number (live nodes):
   longevity-lwt-3h-master-db-node-ed3b168d-1 (54.246.187.202 | 10.0.2.95): 8 shards
   longevity-lwt-3h-master-db-node-ed3b168d-2 (54.74.246.195 | 10.0.3.157): 8 shards
   longevity-lwt-3h-master-db-node-ed3b168d-3 (34.255.209.141 | 10.0.1.148): 8 shards
   longevity-lwt-3h-master-db-node-ed3b168d-4 (34.245.61.12 | 10.0.1.159): 8 shards
   longevity-lwt-3h-master-db-node-ed3b168d-5 (34.244.254.213 | 10.0.1.104): 8 shards
OS (RHEL/CentOS/Ubuntu/AWS AMI): `ami-0abd16bb8d221db89` (aws: eu-west-1)

Test: `longevity-lwt-3h-test`
Test name: `longevity_lwt_test.LWTLongevityTest.test_lwt_longevity`
Test config file(s):
- [longevity-lwt-basic-3h.yaml]([https://github.com/scylladb/scylla-cluster-tests/blob/b80bdcfd983a6f211ea1f654d7eb734eb90c6169/test-cases/longevity/longevity-lwt-basic-3h.yaml](https://www.google.com/url?q=https://github.com/scylladb/scylla-cluster-tests/blob/b80bdcfd983a6f211ea1f654d7eb734eb90c6169/test-cases/longevity/longevity-lwt-basic-3h.yaml&source=gmail-html&ust=1647012885181000&usg=AOvVaw0-dW8HvBpAXnevjFOSnn8G))

**Issue description**

====================================
While running `longevity-lwt-3h-test`, during the `disrupt_add_remove_dc` nemesis we received following warning:

```
Command: 'cqlsh --no-color   --request-timeout=120 --connect-timeout=60  -e "ALTER KEYSPACE system_distributed WITH replication = {\'class\': \'NetworkTopologyStrategy\', \'eu-west\': 3}" 10.0.2.95 9042'

Exit code: 2

Stdout:



Stderr:

<stdin>:1:Warning: schema version mismatch detected; check the schema versions of your nodes in system.local and system.peers.
```

Before the `ALTER KEYSPACE` failure we received a large number of the following errors:
```
022-03-06T07:01:53+00:00 longevity-lwt-3h-master-db-node-ed3b168d-2 !     ERR |  [shard 4] storage_proxy - exception during mutation write to 10.0.3.157: std::runtime_error (Column definition lwt_indicator does not match any column in the query selection)
2022-03-06 07:01:53.536 <2022-03-06 07:01:53.000>: (DatabaseLogEvent Severity.ERROR) period_type=one-time event_id=a6eec539-47a5-4d10-8889-333c2058b187: type=DATABASE_ERROR regex=Exception  line_number=46709 node=longevity-lwt-3h-master-db-node-ed3b168d-3
2022-03-06T07:01:53+00:00 longevity-lwt-3h-master-db-node-ed3b168d-3 !     ERR |  [shard 2] storage_proxy - exception during mutation write to 10.0.1.148: std::runtime_error (Column definition lwt_indicator does not match any column in the query selection)
```

C-s failed with the following errors:

```
2022-03-06 06:55:27.792: (CassandraStressEvent Severity.CRITICAL) period_type=end event_id=477f9704-6689-414b-a493-3903c2edd431 duration=19s: node=Node longevity-lwt-3h-master-loader-node-ed3b168d-3 [54.217.4.101 | 10.0.1.87] (seed: False)
stress_cmd=cassandra-stress write no-warmup cl=ALL n=10000 -schema 'keyspace=keyspace_new_dc replication(strategy=NetworkTopologyStrategy,eu-west=3,eu-west_nemesis_dc=1) compression=LZ4Compressor compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=5 -pop seq=1..10000 -log interval=5
errors:

Stress command completed with bad status 1: Failed to connect over JMX; not collecting these stats
com.datastax.driver.core.exceptions.WriteTime
2022-03-06 06:55:27.791: (CassandraStressEvent Severity.CRITICAL) period_type=end event_id=1a409f0a-6b10-4d98-a792-fe6b40ae2430 duration=50s: node=Node longevity-lwt-3h-master-loader-node-ed3b168d-1 [52.30.239.76 | 10.0.2.202] (seed: False)
stress_cmd=cassandra-stress write no-warmup cl=ALL n=10000 -schema 'keyspace=keyspace_new_dc replication(strategy=NetworkTopologyStrategy,eu-west=3,eu-west_nemesis_dc=1) compression=LZ4Compressor compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=5 -pop seq=1..10000 -log interval=5
errors:

Stress command completed with bad status 1: WARN  06:55:03,194 No schema agreement from live replicas after 10 s. The schema may not be up to da
2022-03-06 06:55:27.794: (CassandraStressEvent Severity.CRITICAL) period_type=end event_id=fab5396a-9614-4891-9d89-8d1a8d826066 duration=18s: node=Node longevity-lwt-3h-master-loader-node-ed3b168d-2 [54.170.179.127 | 10.0.0.117] (seed: False)
stress_cmd=cassandra-stress write no-warmup cl=ALL n=10000 -schema 'keyspace=keyspace_new_dc replication(strategy=NetworkTopologyStrategy,eu-west=3,eu-west_nemesis_dc=1) compression=LZ4Compressor compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=5 -pop seq=1..10000 -log interval=5
errors:

Stress command completed with bad status 1: Failed to connect over JMX; not collecting these stats
com.datastax.driver.core.exceptions.WriteTime
2022-03-06 06:55:48.699: (CassandraStressEvent Severity.CRITICAL) period_type=end event_id=f83eeccc-ede0-44cf-9880-877ca3ebe919 duration=12s: node=Node longevity-lwt-3h-master-loader-node-ed3b168d-1 [52.30.239.76 | 10.0.2.202] (seed: False)
stress_cmd=cassandra-stress read no-warmup cl=ALL n=10000 -schema 'keyspace=keyspace_new_dc compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=5 -pop seq=1..10000 -log interval=5
errors:

Stress command completed with bad status 1: Failed to connect over JMX; not collecting these stats
java.lang.RuntimeException: Failed to execute
2022-03-06 06:56:21.141: (CassandraStressEvent Severity.CRITICAL) period_type=end event_id=e38fc272-b311-4cf6-95fd-ac411e4d678e duration=13s: node=Node longevity-lwt-3h-master-loader-node-ed3b168d-3 [54.217.4.101 | 10.0.1.87] (seed: False)
stress_cmd=cassandra-stress read no-warmup cl=ALL n=10000 -schema 'keyspace=keyspace_new_dc compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=5 -pop seq=1..10000 -log interval=5
errors:

Stress command completed with bad status 1: Failed to connect over JMX; not collecting these stats
java.lang.RuntimeException: Failed to execute
2022-03-06 06:56:22.890: (CassandraStressEvent Severity.CRITICAL) period_type=end event_id=fafc1a1b-ff6f-40eb-8827-0f16aa8e6ce0 duration=15s: node=Node longevity-lwt-3h-master-loader-node-ed3b168d-2 [54.170.179.127 | 10.0.0.117] (seed: False)
stress_cmd=cassandra-stress read no-warmup cl=ALL n=10000 -schema 'keyspace=keyspace_new_dc compression=LZ4Compressor' -port jmx=6868 -mode cql3 native compression=lz4 -rate threads=5 -pop seq=1..10000 -log interval=5
errors:

Stress command completed with bad status 1: Failed to connect over JMX; not collecting these stats
java.lang.RuntimeException: Failed to execute
```
====================================
Restore Monitor Stack command: `$ hydra investigate show-monitor ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9`
Restore monitor on AWS instance using [Jenkins job]([https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-show-monitor/parambuild/?sct_branch=master&test_id=ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9](https://www.google.com/url?q=https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-show-monitor/parambuild/?sct_branch%3Dmaster%26test_id%3Ded3b168d-1b5f-4f8a-8b97-ac94a7731ca9&source=gmail-html&ust=1647012885181000&usg=AOvVaw2I1aBsW9-MqUVdxQvbbt7c))
Show all stored logs command: `$ hydra investigate show-logs ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9`

Test id: `ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9`

Logs:
grafana - [[https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_070322/grafana-screenshot-longevity-lwt-3h-test-scylla-per-server-metrics-nemesis-20220306_070322-longevity-lwt-3h-master-monitor-node-ed3b168d-1.png](https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_070322/grafana-screenshot-longevity-lwt-3h-test-scylla-per-server-metrics-nemesis-20220306_070322-longevity-lwt-3h-master-monitor-node-ed3b168d-1.png)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_070322/grafana-screenshot-longevity-lwt-3h-test-scylla-per-server-metrics-nemesis-20220306_070322-longevity-lwt-3h-master-monitor-node-ed3b168d-1.png%255D(https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_070322/grafana-screenshot-longevity-lwt-3h-test-scylla-per-server-metrics-nemesis-20220306_070322-longevity-lwt-3h-master-monitor-node-ed3b168d-1.png)&source=gmail-html&ust=1647012885181000&usg=AOvVaw0VnVWDRyBxxFUsXgfrMc4V)
critical - [[https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/critical-ed3b168d.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/critical-ed3b168d.log.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/critical-ed3b168d.log.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/critical-ed3b168d.log.tar.gz)&source=gmail-html&ust=1647012885181000&usg=AOvVaw2dxW7h84E1wQGyby4uWsfU)
db-cluster - [[https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/db-cluster-ed3b168d.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/db-cluster-ed3b168d.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/db-cluster-ed3b168d.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/db-cluster-ed3b168d.tar.gz)&source=gmail-html&ust=1647012885181000&usg=AOvVaw0t5HMVbl8Pt7lC2fPR_KYU)
debug - [[https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/debug-ed3b168d.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/debug-ed3b168d.log.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/debug-ed3b168d.log.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/debug-ed3b168d.log.tar.gz)&source=gmail-html&ust=1647012885181000&usg=AOvVaw32LbUwU0ceCqELnGpWRtd0)
email_data - [[https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/email_data-ed3b168d.json.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/email_data-ed3b168d.json.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/email_data-ed3b168d.json.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/email_data-ed3b168d.json.tar.gz)&source=gmail-html&ust=1647012885181000&usg=AOvVaw1hPFWihZzPx0dxcLuOPMRz)
error - [[https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/error-ed3b168d.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/error-ed3b168d.log.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/error-ed3b168d.log.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/error-ed3b168d.log.tar.gz)&source=gmail-html&ust=1647012885181000&usg=AOvVaw3Bf2qPiqrDNaTtz5RahPCu)
event - [[https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/events-ed3b168d.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/events-ed3b168d.log.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/events-ed3b168d.log.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/events-ed3b168d.log.tar.gz)&source=gmail-html&ust=1647012885181000&usg=AOvVaw3pbL0tjo5opfw014g4kmQB)
left_processes - [[https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/left_processes-ed3b168d.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/left_processes-ed3b168d.log.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/left_processes-ed3b168d.log.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/left_processes-ed3b168d.log.tar.gz)&source=gmail-html&ust=1647012885181000&usg=AOvVaw2BIhNlKP1i2XVaa22MJjtm)
loader-set - [[https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/loader-set-ed3b168d.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/loader-set-ed3b168d.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/loader-set-ed3b168d.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/loader-set-ed3b168d.tar.gz)&source=gmail-html&ust=1647012885181000&usg=AOvVaw06GrAvUGoAdMrkoz6oDp_0)
monitor-set - [[https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/monitor-set-ed3b168d.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/monitor-set-ed3b168d.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/monitor-set-ed3b168d.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/monitor-set-ed3b168d.tar.gz)&source=gmail-html&ust=1647012885181000&usg=AOvVaw1r6D2xl8L1MzWINuEtdrOg)
normal - [[https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/normal-ed3b168d.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/normal-ed3b168d.log.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/normal-ed3b168d.log.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/normal-ed3b168d.log.tar.gz)&source=gmail-html&ust=1647012885181000&usg=AOvVaw3JhifkCyHmGdSd4V19XxKg)
output - [[https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/output-ed3b168d.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/output-ed3b168d.log.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/output-ed3b168d.log.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/output-ed3b168d.log.tar.gz)&source=gmail-html&ust=1647012885181000&usg=AOvVaw01vICdb8rCqvca2xUt2gz8)
event - [[https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/raw_events-ed3b168d.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/raw_events-ed3b168d.log.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/raw_events-ed3b168d.log.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/raw_events-ed3b168d.log.tar.gz)&source=gmail-html&ust=1647012885181000&usg=AOvVaw3E7SLeCJu89o7DhTa7aoc8)
sct - [[https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/sct-ed3b168d.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/sct-ed3b168d.log.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/sct-ed3b168d.log.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/sct-ed3b168d.log.tar.gz)&source=gmail-html&ust=1647012885181000&usg=AOvVaw2IjYYMvdcmU35odNoasUhI)
summary - [[https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/summary-ed3b168d.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/summary-ed3b168d.log.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/summary-ed3b168d.log.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/summary-ed3b168d.log.tar.gz)&source=gmail-html&ust=1647012885181000&usg=AOvVaw3ayV_F5Mm-61gxndVpS6NX)
warning - [[https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/warning-ed3b168d.log.tar.gz](https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/warning-ed3b168d.log.tar.gz)](https://www.google.com/url?q=https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/warning-ed3b168d.log.tar.gz%255D(https://cloudius-jenkins-test.s3.amazonaws.com/ed3b168d-1b5f-4f8a-8b97-ac94a7731ca9/20220306_071636/warning-ed3b168d.log.tar.gz)&source=gmail-html&ust=1647012885181000&usg=AOvVaw2Nwbt9_n5im9a4sMOqIukj)

[Jenkins job URL]([https://jenkins.scylladb.com/job/scylla-master/job/longevity/job/longevity-lwt-3h-test/330/](https://www.google.com/url?q=https://jenkins.scylladb.com/job/scylla-master/job/longevity/job/longevity-lwt-3h-test/330/&source=gmail-html&ust=1647012885181000&usg=AOvVaw03POw2d6FiZfugCQBrOHEM))

**Comment by juliayakovlev** [comment]
Reproduced

Nemesis `ToggleTableIcs`:
```
< t:2022-09-30 02:16:17,425 f:file_logger.py  l:101  c:sdcm.sct_events.file_logger p:INFO  > 2022-09-30 02:16:17.424: (DisruptionEvent Severity.NORMAL) period_type=begin event_id=7f258d43-89a6-4db1-9af9-1e8354a2b61f: nemesis_name=ToggleTableIcs target_node=Node longevity-lwt-500G-3d-2022-2-db-node-4a98679f-2 [44.195.32.109 | 10.12.1.58] (seed: False)
```

Alter MV with "IncrementalCompactionStrategy":
```
< t:2022-09-30 02:16:22,871 f:nemesis.py      l:2181 c:sdcm.nemesis         p:DEBUG > sdcm.nemesis.SisyphusMonkey: Toggle table ICS query to execute: ALTER MATERIALIZED VIEW  cqlstress_lwt_example.blogposts_not_updated_lwt_indicator WITH compaction = {'class': 'IncrementalCompactionStrategy', 'bucket_high': 1.68, 'bucket_low': 0.69, 'min_sstable_size': 51, 'min_threshold': 3, 'max_threshold': 50, 'sstable_size_in_mb': 684};

< t:2022-09-30 02:16:25,374 f:base.py         l:142  c:RemoteLibSSH2CmdRunner p:DEBUG > Command "cqlsh --no-color   --request-timeout=120 --connect-timeout=60  -e "ALTER MATERIALIZED VIEW  cqlstress_lwt_example.blogposts_not_updated_lwt_indicator WITH compaction = {'class': 'IncrementalCompactionStrategy', 'bucket_high': 1.68, 'bucket_low': 0.69, 'min_sstable_size': 51, 'min_threshold': 3, 'max_threshold': 50, 'sstable_size_in_mb': 684};" 10.12.1.58 9042" finished with status 0
```

Altering MV and base table:
```
< t:2022-09-30 02:16:24,082 f:db_log_reader.py l:113  c:sdcm.db_log_reader   p:DEBUG > 2022-09-30T02:16:24+00:00 longevity-lwt-500G-3d-2022-2-db-node-4a98679f-1     !INFO | scylla[7805]:  [shard  0] schema_tables - Altering cqlstress_lwt_example.blogposts id=73f6f380-3fe8-11ed-b0b0-db916f78562b version=e03be5db-1538-33b5-89e3-98ea87366f36
< t:2022-09-30 02:16:24,083 f:db_log_reader.py l:113  c:sdcm.db_log_reader   p:DEBUG > 2022-09-30T02:16:24+00:00 longevity-lwt-500G-3d-2022-2-db-node-4a98679f-1     !INFO | scylla[7805]:  [shard  0] schema_tables - Altering cqlstress_lwt_example.blogposts_not_updated_lwt_indicator id=795c0630-3fe8-11ed-9831-768e3e3200cc version=24520007-b3b9-3401-a621-06375b29d808
```

Schema agreement has been completed:
```
< t:2022-09-30 02:16:36,729 f:health_checker.py l:254  c:sdcm.utils.health_checker p:DEBUG > Schema agreement has been completed on all nodes
```

But started to report about storage_proxy exception during mutation write
```
< t:2022-09-30 02:16:37,226 f:db_log_reader.py l:113  c:sdcm.db_log_reader   p:DEBUG > 2022-09-30T02:16:37+00:00 longevity-lwt-500G-3d-2022-2-db-node-4a98679f-1      !ERR | scylla[7805]:  [shard  3] storage_proxy - exception during mutation write to 10.12.1.240: std::runtime_error (Column definition lwt_indicator does not match any column in the query selection)
```

And cassandra stress `WriteFailureException`:
```
< t:2022-09-30 02:16:37,003 f:base.py         l:228  c:RemoteLibSSH2CmdRunner p:DEBUG > com.datastax.driver.core.exceptions.WriteFailureException: Cassandra failure during write query at consistency QUORUM (2 responses were required but only 0 replica responded, 2 failed)
```

## Installation details

Kernel Version: 5.15.0-1019-aws
Scylla version (or git commit hash): `2022.2.0~rc2-20220919.75d087a2b75a` with build-id `463f1a57b82041a6c6b6441f0cbc26c8ad93091e`

Cluster size: 4 nodes (i3.4xlarge)

Scylla Nodes used in this run:
- longevity-lwt-500G-3d-2022-2-db-node-4a98679f-7 (100.27.38.242 | 10.12.2.247) (shards: 14)
- longevity-lwt-500G-3d-2022-2-db-node-4a98679f-6 (3.218.248.198 | 10.12.3.36) (shards: 14)
- longevity-lwt-500G-3d-2022-2-db-node-4a98679f-5 (18.213.247.208 | 10.12.2.105) (shards: 14)
- longevity-lwt-500G-3d-2022-2-db-node-4a98679f-4 (3.234.206.161 | 10.12.2.68) (shards: 14)
- longevity-lwt-500G-3d-2022-2-db-node-4a98679f-3 (3.236.36.209 | 10.12.0.98) (shards: 14)
- longevity-lwt-500G-3d-2022-2-db-node-4a98679f-2 (44.195.32.109 | 10.12.1.58) (shards: 14)
- longevity-lwt-500G-3d-2022-2-db-node-4a98679f-1 (3.237.96.30 | 10.12.1.240) (shards: 14)


OS / Image: `ami-0b6ff8cdcbe0cb88a` (aws: us-east-1)

Test: `longevity-lwt-500G-3d-test`
Test id: `4a98679f-02ad-4c38-a717-833dd12453de`
Test name: `enterprise-2022.2/longevity/longevity-lwt-500G-3d-test`
Test config file(s):

- [longevity-lwt-500G-3d.yaml](https://github.com/scylladb/scylla-cluster-tests/blob/9818d76f6d1584c8d4652429a6aa1c616b6bb3c2/test-cases/longevity/longevity-lwt-500G-3d.yaml)



- Restore Monitor Stack command: `$ hydra investigate show-monitor 4a98679f-02ad-4c38-a717-833dd12453de`
- Restore monitor on AWS instance using [Jenkins job](https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-show-monitor/parambuild/?test_id=4a98679f-02ad-4c38-a717-833dd12453de)
- Show all stored logs command: `$ hydra investigate show-logs 4a98679f-02ad-4c38-a717-833dd12453de`

## Logs:
db-cluster - https://cloudius-jenkins-test.s3.amazonaws.com/4a98679f-02ad-4c38-a717-833dd12453de/20221002_145214/db-cluster-4a98679f.tar.gz


[Jenkins job URL](https://jenkins.scylladb.com/job/enterprise-2022.2/job/longevity/job/longevity-lwt-500G-3d-test/4/)

**Comment by DoronArazii** [comment]
@juliayakovlev can you please share if this a regression

**Comment by DoronArazii** [comment]
@juliayakovlev ping

/Cc @roydahan

**Comment by juliayakovlev** [comment]
> @juliayakovlev can you please share if this a regression

@DoronArazii as I found it happened in 5.0, 5.1, 2022.1 and 2022.2

**Comment by DoronArazii** [comment]
@avikivity is it something we need to handle with in 5.1?

**Comment by kostja** [comment]
see also https://github.com/scylladb/scylladb/issues/11542 
Looks like pretty much the same issue.

**Comment by DoronArazii** [comment]
@kostja any progress here?
I see that https://github.com/scylladb/scylladb/issues/11542 was closed.

**Comment by kostja** [comment]
https://github.com/scylladb/scylladb/pull/11986 contains the fix.

**Comment by DoronArazii** [comment]
The fix is in master since last Nov - part of 5.2.0-rc0

Closing.
